platform-hub-api/spec/factories/kubernetes_tokens.rb

```diff
@@ -39,7 +43,7 @@
 
       trait :user_is_not_member_of_project do
         after(:build) do |token, _|
-          ProjectMembership.where(project_id: token.project_id, user_id: token.tokenable.user_id).delete_all
+          ProjectMembership.where(project_id: token.project_id, user_id: token.tokenable.user_id).map(&:destroy)
```

Comment:
or just 
```ruby
ProjectMembership.destroy_all(project_id: token.project_id, user_id: token.tokenable.user_id)
```

Reply:
I was under the impression that `destroy_all` would not execute any `after_destroy` callbacks on the record being deleted.

Reply:
I think it does but it's a minor thing. Can leave as is.